internal/compliance/snapshot_poller/pollers/aws/poll.go

```diff
@@ -374,7 +374,8 @@ func singleResourceScan(
 
 	if err != nil {
 		// Check for region ignoreList error
-		if err == err.(*RegionIgnoreListError) {
+		var e *RegionIgnoreListError
+		if errors.As(err, &e) {
```

Comment:
The `&e` is now a double pointer (`**RegionIgnoreListError`). That might work, but probably not what you want.

```suggestion
		var e RegionIgnoreListError
		if errors.As(err, &e) {
```

Reply:
So upon second review, it appears that I need to use a double pointer here. Following the [Go Blog](https://blog.golang.org/go1.13-errors): 

> `errors.As` will panic if the target is not a non-nil pointer to either a type that implements error, or to any interface type. 

Instantiating `e` as `var e RegionIgnoreListError` causes `errors` to complain: `*target must be interface or implement error`